core/src/apps/webauthn/fido2.py

```
@@ -1812,7 +1812,7 @@ def cbor_get_assertion_hmac_secret(cred: Credential, hmac_secret: dict) -> bytes
         raise CborError(_ERR_INVALID_LEN)
 
     # Compute the ECDH shared secret.
-    ecdh_result = nist256p1.multiply(KEY_AGREEMENT_PRIVKEY, b"\04" + x + y)
+    ecdh_result = nist256p1.multiply(KEY_AGREEMENT_PRIVKEY, b"\04" + x + y, True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)